### DISCUSSION TITLE
Bus Stop Name on top of display

### DISCUSSION THREAD

**Discussion by JoshX21** [jfoot/Bus-Departure-Board#30] [Ideas]
Would it be possible to display the Bus Stop Name at the top of the display for the National Buses script?

**Comment by jfoot** [comment]
Hi @JoshX21, the National Buses program draws effectively 4 rows of text, three departure details and the time at the bottom. I could relatively easily add the stop name at the bottom in replacement of the time? Although, to add it at the top is more complicated and requires resizing all other rows to squeeze it in. I'll keep this request in mind for the future, but currently don't have the time to implement it (just started a new job). I'll see if I can come back to it later in the year.

![image](https://user-images.githubusercontent.com/27645114/192093324-a9a4ee84-c36e-4b3f-bb36-e9174297d6f2.png)
